Added UI option to set tag_ranges

Alopalao opened the pull request:
Closes #50 

### Summary

Added two `k-accordion-item` for `available_tags` and `tag_ranges` from `Interface`.
![Screenshot from 2023-10-05 17-16-00](https://github.com/kytos-ng/ui/assets/55767214/8680abc2-45fe-495e-b37e-486cd1b813c8)
 Functionality works but UI style can change.
This PR needs commits from `topology` PR [#168](https://github.com/kytos-ng/topology/pull/168) and `kytos` PR [#421](https://github.com/kytos-ng/kytos/pull/421)

### Localt Test
- Tried setting applicable new tag_ranges
- Tried setting un-applicable new tag_ranges
- Modified so API request fail and looked at the messages.

viniarck commented on src/kytos/interfaceInfo.vue:
```diff
@@ -21,7 +21,50 @@
+          </table>
+        </div>
+      </k-accordion-item>
+      <k-accordion-item title="Tag range">
```
```suggestion
      <k-accordion-item title="Tag ranges">
```

Trivial detail, but if you could pluralize here that'd be great.

Alopalao replied:
Fixed

viniarck commented on src/kytos/interfaceInfo.vue:
```diff
@@ -21,7 +21,50 @@
+            <tbody>
+              <tr v-for="(value, key) in this.available_tags">
+                <td >{{key}}</td>
+                <td >{{value}}</td>
```
Aldo, can you try to align the text left? I have an impression that the range representation with text centered becomes a bit harder to read, for instance, check out how these squared brackets ended up:

![20231006_130140](https://github.com/kytos-ng/ui/assets/1010796/a9be5b93-1363-4c0c-834c-792856deb133)

Same comment for tag ranges `td` entries below

Alopalao replied:
Changed align to left

viniarck reviewed (commented):
Excellent to have the ability to set and display `tag_ranges` and display `available_tags` @Alopalao. 

I asked a minor styling change. Also, don't forget to update the changelog. Let me know also include @rmotitsuki in the review in case he wants to review too.

viniarck reviewed (approved):
Excellent PR @Alopalao. I'll leave this PR pre-approved, but don't forget to update the changelog. Great display/formatting when showing errors too. Let's leave this PR for a few days in case any other team member wants to review, but it'll also land soon with your other related PRs (although the other 4 ones can land first since they've been longer sitting in code review). 

![20231006_152654](https://github.com/kytos-ng/ui/assets/1010796/faa6b764-9652-41c1-b6b2-c38f8ef34ad6)

![20231006_152732](https://github.com/kytos-ng/ui/assets/1010796/f74460ee-b45f-4873-abdc-9b46d852b4d7)

Alopalao commented:
Commit [d98c1fe6ea2a44eb54596bb4091adbbfd1eb7d7e](https://github.com/kytos-ng/ui/pull/51/commits/d98c1fe6ea2a44eb54596bb4091adbbfd1eb7d7e)

- Updated Changelog
- Deleted unnecessary code

viniarck commented:
> Commit [d98c1fe6ea2a44eb54596bb4091adbbfd1eb7d7e](https://github.com/kytos-ng/ui/pull/51/commits/d98c1fe6ea2a44eb54596bb4091adbbfd1eb7d7e)
> 
> * Updated Changelog
> * Deleted unnecessary code

Looks good, Aldo. 

Let's merge this PR by tomorrow, let's leave one extra day in case @rmotitsuki might still want to review.
